### PR TITLE
Adding pylibdmtx

### PR DESCRIPTION
**Pull request by Ryan-Dmello**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pylibdmtx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "pylibdmtx" %}
+{% set version = "0.1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: "https://github.com/NaturalHistoryMuseum/pylibdmtx.git"
+  git_rev: "v{{ version }}"
```
Comment (beckermr) [inline recipes/pylibdmtx/meta.yaml:10]: We strongly discourage git urls. Can you ping upstream for a release?

File: recipes/pylibdmtx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - python
+    - pip
+  run:
+    - python
+    - Pillow
+
+test:
+  requires:
+    - pytest
+    - numpy
```
Comment (beckermr) [inline recipes/pylibdmtx/meta.yaml:27]: Unless you are running the tests, you don't need these

File: recipes/pylibdmtx/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - Pillow
```
Comment (beckermr) [inline recipes/pylibdmtx/meta.yaml:22]: Do you need numpy here? Does the package import numpy directly?

File: recipes/pylibdmtx/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: "https://github.com/NaturalHistoryMuseum/pylibdmtx.git"
+  git_rev: "v{{ version }}"
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (beckermr) [inline recipes/pylibdmtx/meta.yaml:14]: Can this recipe be noarch python?

Comment (Ryan-Dmello) [reply recipes/pylibdmtx/meta.yaml]: Making it noarch gives below issue, hence not making noarch:
[noarch_python] Noarch package contains binary script: write_datamatrix.exe

File: recipes/pylibdmtx/meta.yaml (hunk 3 above)
Comment (Ryan-Dmello) [reply recipes/pylibdmtx/meta.yaml:17]: Removed

File: recipes/pylibdmtx/meta.yaml (hunk 2 above)
Comment (Ryan-Dmello) [reply recipes/pylibdmtx/meta.yaml]: Removed

File: recipes/pylibdmtx/meta.yaml (hunk 1 above)
Comment (Ryan-Dmello) [reply recipes/pylibdmtx/meta.yaml]: Updated not use use git_url. Added build.sh and bld.bat to install from wheel files available on pypi.org

**Pull request merged**